MapboxCoreNavigation/NativeRouteController.swift

```diff
@@ -0,0 +1,79 @@
+import Foundation
+import MapboxNavigationNative
+import MapboxDirections
+
+
+public class NativeRouteController: RouteController {
```

Comment:
This public class doesn’t bridge to Objective-C; should it?

Reply:
All the route controllers are “native”, least of all this one. Since this class is public, can we call it something more descriptive and less misleading? How about `CrossPlatformRouteController` or `PortableRouteController`?

Reply:
How about renaming the old one to LegacyRouteController and Native to just RouteController?

Reply:
What would the migration path look like for existing clients of the SDK that are using RouteController directly? As long as there’s more than one route controller, I think it would be understandable to qualify the class with an additional word. If we plan to remove the existing RouteController entirely in this release, that would be a different story (with implications for the changelog).